spec/domain/people/neuanmeldungen/approve_spec.rb

```diff
@@ -15,13 +15,13 @@
   let(:neuanmeldungen_sektion) { groups(:bluemlisalp_neuanmeldungen_sektion) }
   let(:neuanmeldungen_nv) { groups(:bluemlisalp_neuanmeldungen_nv) }
 
-  def create_role(beitragskategorie)
+  def create_role(beitragskategorie, person: nil)
```

Comment:
```suggestion
  def create_role(beitragskategorie, person: Fabricate(:person, birthday: 20.years.ago, sac_family_main_person: true))
```
So müsste das Fabricate glaub der default wert sein dann kannst du unten das `||` lassen